src/applications/vaos/services/mocks/var/direct_booking_eligibility_criteria.json

```diff
@@ -380,7 +380,6 @@
           {
             "id": "502",
             "typeOfCare": "Outpatient Mental Health",
-            "patientHistoryRequired": "No",
```

Comment:
Updated our mock data so that mental health is always the single facility scenario